src/pages/messaging/agent-chat-api/index.mdx

```diff
@@ -1089,7 +1089,7 @@ https://api.livechatinc.com/v3.4/agent/action/add_user_to_chat \
     "user_id": "smith@example.com",
     "user_type": "agent",
     "visibility": "all",
-    "require_active_thread": true
+    "ignore_requester_presence": true
```

Comment:
Issue reported by Aleksander. I confirmed with Paweł Żuczkowski from API that in v3.4 and v3.5, the method requires the requester to be present on the list of chat users so that I can override it with the `ignore_requester_presence` param. Like we thought - a copy-and-paste mistake.